apps/admin/controllers/settings/index.rb

```diff
@@ -0,0 +1,11 @@
+module Admin::Controllers::Settings
+  class Index
+    include Admin::Action
+
+    expose :settings
+
+    def call(params)
+      @settings = SettingRepository.new.for_display.first
```

Comment:
not sure that we need to call `#first` method here. Let's move it to repository method. WDYT?

Reply:
Sure much better actually 🎉